platform.md

```diff
@@ -146,94 +147,70 @@ The following is a non-exhaustive list of terms defined in the [OCI Distribution
-- The image config's `Label` field has the label `io.buildpacks.stack.released` set to the release date of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.description` set to the description of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.metadata` set to additional metadata related to the stack.
+- The image config's `Label` field has the label `io.buildpacks.base.maintainer` set to the name of the image maintainer.
+- The image config's `Label` field has the label `io.buildpacks.base.homepage` set to the homepage of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.released` set to the release date of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.description` set to the description of the image.
+- The image config's `Label` field has the label `io.buildpacks.base.metadata` set to additional metadata related to the image.
 
 ### Run Image
 
+A typical run image might determine:
+* The OS distro or distroless OS in the launch environment.
+* OS packages installed in the launch environment.
+* Trusted CA certificates in the launch environment.
+* The default user in the run environment.
+
 The platform MUST ensure that:
 
-- The image config's `Label` field has the label `io.buildpacks.stack.id` set to the stack ID.
-- The image config's `Label` field has the label `io.buildpacks.stack.mixins` set to a JSON array containing mixin names for each mixin applied to the image.
 - The image config's `Env` field has the environment variable `PATH` set to a valid set of paths or explicitly set to empty (`PATH=`).
 
 The platform SHOULD ensure that:
 
 - The image config's `User` field is set to a user with a **DIFFERENT** user [†](README.md#operating-system-conventions)UID/[‡](README.md#operating-system-conventions)SID as the build image.
-- The image config's `Label` field has the label `io.buildpacks.stack.maintainer` set to the name of the stack maintainer.
-- The image config's `Label` field has the label `io.buildpacks.stack.homepage` set to the homepage of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.distro.name` set to the name of the stack's OS distro.
-- The image config's `Label` field has the label `io.buildpacks.stack.distro.version` set to the version of the stack's OS distro.
-- The image config's `Label` field has the label `io.buildpacks.stack.released` set to the release date of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.description` set to the description of the stack.
-- The image config's `Label` field has the label `io.buildpacks.stack.metadata` set to additional metadata related to the stack.
+- The image config's `Label` field has the label `io.buildpacks.id` set to the target ID (e.g., "minimal") of the run image.
```

Comment:
what is "minimal" here? (I'm not understanding the example).

Reply:
should it be tiny/base/full [as in paketo](https://hub.docker.com/r/paketobuildpacks/builder)?

Reply:
@natalieparellano do we use `io.buildpacks.run` anywhere today? Why `run` over `base`?

Reply:
To my knowledge, `io.buildpacks.run` isn't used anywhere today. I like it better than `base` because base is ambiguous - which base? The build-time base, or runtime base, or both? IMO it leads to an ambiguous coupling between the build- and runtime base images which we are trying to avoid with the removal of the concept of "stack". Though, to be fair, they are always going to be tied together to some degree. I won't die on this hill, just putting it out there...

Reply:
I don't have a strong preference besides we aren't doing anything with `io.buildpacks.run` today. Are there other things we should put there or would you like to just start with this? What I do care about is it remaining on `io.buildpacks.id`, so as long as we change that, I'm :+1: .

Reply:
Or how about `io.buildpacks.target.id` (for parity with `CNB_TARGET_ID`)? Or `io.buildpacks.target`?

Reply:
@hone I don't feel strongly about this. It's hard to provide guidance about a feature that's not being used yet. I think in the future we might be able to give some more useful examples.

Reply:
@natalieparellano I think the counterpoint, we're adding this field for a reason and with intention. From what I seem to be gathering so far is it being used in place of `CNB_STACK_ID`. Especially, since we're providing this to Buildpack Authors in the Buildpack API, Buildpacks can use it to change logic based on this value. From WG yesterday, what happens if there's different base/builder images that use the same target id?

Reply:
Looking at other parts of the spec, when we use "identifier", it usually is "machine readable" and has restrictions on what we allow in it:


* MUST NOT be identical to any other ID when using a case-insensitive comparison.
* MUST only contain numbers, letters, and the characters ., /, and -.
* SHOULD use reverse domain name notation to avoid name collisions.

While we use "name" to have something human readable without any restrictions.

Does it make sense to apply similar constraints here?

Reply:
Sounds good - please see bc1e753

Reply:
Cool. I've further updated some of my thoughts on the target ID over in https://github.com/buildpacks/community/discussions/230; maybe we can discuss in the WG meeting on Thursday?

Reply:
This is great. Thanks for working with me on this. We can do the rest of the guidance I'm looking for on "what" to use target id for on the documentation side.